### DISCUSSION TITLE
USP-Strip Outlet Control

### DISCUSSION THREAD

**Discussion by JSylvia007** [Art-of-WiFi/UniFi-API-client#183] [Q&A] [answered]
Has anyone used this API to control the outlets of either the USP-Strip or SmartPower PDU Pro (USP-PDU-Pro)?  I'm moving away from Raritan power devices because all of the legacy ones in my homelab are now so out of date that I can't access them via the web.  I do have some scripts however that force automated power actions, and I would love to know if someone has already figured this out.

**Comment by JSylvia007** [comment, marked as answer]
I am a jackass...

https://github.com/Art-of-WiFi/UniFi-API-client/blob/master/examples/modify_smartpower_pdu_outlet.php

**Comment by malle-pietje** [comment]
😉
